libs/execution_context/ebpf_maps.c

```diff
@@ -1464,3 +1492,47 @@ ebpf_map_get_info(
     *info_size = sizeof(*info);
     return EBPF_SUCCESS;
 }
+
+static void
+_ebpf_map_cancel_context(_In_ void* cancel_context)
```

Comment:
```suggestion
_ebpf_map_cancel_context(_In_ _Post_invalid_ void* cancel_context)
```
since it's freed in line 1504